tests/unit/stats/test_view_data.py

```diff
@@ -15,9 +15,10 @@
 import unittest
 import mock
 from datetime import datetime
-from opencensus.stats import aggregation as aggregation_module
-from opencensus.stats import view as view_module
 from opencensus.stats import view_data as view_data_module
+from opencensus.stats import view as view_module
+from opencensus.stats import aggregation as aggregation_module
```

Comment:
Could you sort the imports alphabetically?

Reply:
Done